daemons/gptp/common/common_port.cpp

```diff
@@ -467,8 +467,18 @@ bool CommonPort::processStateChange( Event e )
 
 bool CommonPort::processSyncAnnounceTimeout( Event e )
 {
+	// We're Grandmaster, set grandmaster info to me
+	ClockIdentity clock_identity;
+	unsigned char priority1;
+	unsigned char priority2;
+	ClockQuality clock_quality;
+
+	Timestamp system_time;
+	Timestamp device_time;
+	uint32_t local_clock, nominal_clock_rate;
+
 	// Nothing to do
-	if( clock->getPriority1() != 255 )
+	if( clock->getPriority1() == 255 )
```

Comment:
The logic here is that if priority1 == 255, we are never going to become GM so we don't care if the SyncAnnounceTimeout occurred?